CONTRIBUTING.md

```diff
@@ -0,0 +1,135 @@
+
+Samples:
+
+Commit message with no body:
+
+```
+fix: add tty & fix typo on docker-compose
+```
+
+Commit message with scope:
+
+```
+feat(agent): return error message to serial
+```
+
+#### Revert
+
+If the commit reverts a previous commit, it should begin with `revert:` , followed by the header of the reverted commit. In the body it should say: `This reverts commit <hash>.`, where the hash is the SHA of the commit being reverted.
+
+#### Type
+
+Must be one of the following:
+
+- **build**: Changes that affect the build system or external dependencies (example scopes: cargo, make...)
+- **chore**: Some housekeeping activity
+- **ci**: Changes to our CI configuration files and scripts
+- **docs**: Documentation only changes
+- **feat**: A new feature
+- **fix**: A bug fix
+- **perf**: A code change that improves performance
+- **refactor**: A code change that neither fixes a bug nor adds a feature
+- **revert**: A commit revert
+- **test**: Adding missing tests or correcting existing tests
+
+#### Scope
+
+The scope should be the name of the related Lambdo component if applicable.
+
+The following is the list of supported scopes:
+
+- agent
+- api
+- initramfs
+- shared
+
+The scope of application is optional and only needs to be specified if the modifications concern one of the components listed above.
+
+#### Subject
+
+The subject contains a succinct description of the change:
+
+- use the imperative, present tense: `change` not `changed` nor `changes`
+- don't capitalize the first letter
+- no dot (.) at the end
+
+#### Body
+
+Just as in the subject, use the imperative, present tense: `change` not `changed` nor `changes`. The body should include the motivation for the change and contrast this with previous behavior.
+
+#### Footer
+
+The footer should contain any information about Breaking Changes and is also the place to [reference GitHub issues that the commit closes](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).
+
+**Breaking Changes** should start with the word `BREAKING CHANGE:` with a space or two newlines. The rest of the commit message is then used for this.
+
+#### Sign-off
+
+Please consider signing the commit message at least with `Signed-Off-By`. This is a way to certify that you wrote the code or otherwise have the right to pass it on as an open-source patch. The process is simple: if you can certify the [Developer's Certificate of Origin](https://developercertificate.org/) (DCO).
+
+To perform a sign-off with `git`, use :
+
+```bash
+git commit -s #(or --signoff)
+```
+
+## Get help
+
+If you have questions about the contribution process, please start a [GitHub discussion](https://github.com/faast-rt/lambdo/discussions) or send your question to the contact email address: [contact@faast-rt.com](mailto://contact@faast-rt.com).
```

Comment:
Do we really need all this text about conventional commits ? This is 2/3 of our contributing. We can have a small bullet point list to recap what it is about, but the reader is expected to read the specs if they don't know it.

Reply:
I prefer to have something that is clearly defined than a reference to another convention that can change at any time without realizing it.